GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockGitHubUpgrader.cs

```diff
@@ -56,6 +61,11 @@ public void ResetFailedAction()
             this.failActionTypes = ActionType.Invalid;
         }
 
+        public bool RunInstallerInvoked()
+        {
+            return this.InstallerArgs.Count > 0;

```

Comment:
This seems like a bit indirect way to check this... on 1st glance, I wasn't sure what `InstallerArgs` had to do with checking whrether an installer was invoked. Maybe a comment that InstallerArgs captures the args from each of the install commands, or tweak the variable name?

Reply:
right. replaced it with a more obvious field - `InstallerExeLaunched`.